streams/src/main/java/org/apache/kafka/streams/processor/internals/metrics/StreamsMetricsImpl.java

```diff
@@ -54,6 +54,9 @@
     public static final String PROCESSOR_NODE_METRICS_GROUP = "stream-processor-node-metrics";
     public static final String PROCESSOR_NODE_ID_TAG = "processor-node-id";
 
+    public static final String CLIENT_METRICS_GROUP = "stream-metrics";
```

Comment:
"client metrics group" seems like a weird name for this. What do you think about `STREAMS_METRICS_GROUP` or `TOP_LEVEL_METRICS_GROUP` or `INSTANCE_METRICS_GROUP`?

Reply:
I like `INSTANCE_METRICS_GROUP`, will do so.